app/src/main/res/layout/screen_patient_summary.xml

```diff
@@ -203,32 +203,32 @@
     android:paddingTop="@dimen/spacing_8"
     android:paddingBottom="@dimen/spacing_8">
 
-    <com.google.android.material.button.MaterialButton
-      android:id="@+id/doneButton"
-      style="?attr/materialButtonStyle"
+    <org.simple.clinic.widgets.ProgressMaterialButton
+      android:id="@+id/doctorButton"
       android:layout_width="0dp"
       android:layout_height="wrap_content"
       android:layout_marginStart="@dimen/spacing_8"
       android:layout_marginEnd="@dimen/spacing_8"
       android:layout_weight="1"
-      android:text="@string/patientsummary_finish_saving_patient"
+      android:text="@string/patientsummary_contact_doctor"
+      android:textAllCaps="true"
       android:textAppearance="@style/Clinic.V2.TextAppearance.Button1"
-      app:icon="@drawable/ic_done_24dp"
-      app:iconGravity="textStart" />
+      android:visibility="gone"
+      app:backgroundTint="@color/contact_doctor_button"
+      app:iconTint="@android:color/white" />
```

Comment:
Since we are no longer using icon for this button, we can remove the icon tint attr.